src/api/controllers/authController.js

```diff
@@ -0,0 +1,66 @@
+import bcrypt, { hashSync, genSaltSync } from 'bcrypt';
+import models from '../models/index';
+import generateToken from '../../helpers/tokens/generate.token';
+import sendResult from '../../helpers/results/send.auth';
+
+const { User } = models;
```

Comment:
I think the users model should be in plural and shouldn't start with a capital letter. I would like to know the opinion of @MemunaHaruna

Reply:
`User` is fine, in my opinion.

Reply:
Oh. Great. I will also follow that.